add-domain

```diff
@@ -1039,6 +1043,3 @@ youareanidiot.cc
 youthful-engelbart.34-125-197-109.plesk.page
 zc11m.csb.app
 zttmct-tlemp.web.app
-disocrds.gift
-steamcommunitvs.com
-disceord.gift
```

Comment:
These three domains needs to stay, they are potential active

![image](https://user-images.githubusercontent.com/44526987/156900523-8e6b7e24-3902-4502-a853-55d103e58987.png)